addons/base_import_module/tests/test_import_module.py

```diff
@@ -57,3 +57,63 @@ def test_import_and_uninstall_module(self):
 
         asset_data = self.env['ir.model.data'].search([('model', '=', 'ir.asset'), ('res_id', '=', asset.id)])
         self.assertEqual(len(asset_data), 0)
+
+    def test_import_and_update_module(self):
```

Comment:
this test isn't failing if I comment the code change at https://github.com/odoo/odoo/pull/91707/files#diff-3d414e7f4df2b7703572278aa6906de5dec69481842c0ea364eca2e20e6f6e3dR103 so you are not testing your change correctly.

Reply:
Updated. Please check~